Assets/MixedRealityToolkit.Services/InputSystem/FocusProvider.cs

```diff
@@ -1019,17 +1025,33 @@ private static void QueryScene(IMixedRealityPointer pointer, IMixedRealityRaycas
                         }
                         break;
                     case SceneQueryType.SphereOverlap:
-                        Collider[] colliders = UnityEngine.Physics.OverlapSphere(pointer.Rays[i].Origin, pointer.SphereCastRadius, ~UnityEngine.Physics.IgnoreRaycastLayer);
+                        // Set up our results array
+                        if (colliders == null)
+                        {
+                            colliders = new Collider[maxQuerySceneResults];
+                        }
+                        else if (colliders.Length != maxQuerySceneResults)
+                        {
+                            Array.Resize<Collider>(ref colliders, maxQuerySceneResults);
+                        }
```

Comment:
Is the plan here to make it so that maxQuerySceneResults is changeable at runtime as well? Otherwise this code will never be invoked/useful.

Reply:
My guess is it will never change but that's not totally settled: #5012 
So I figured better safe than sorry.